src/ajax/map_variants.php

```diff
@@ -198,6 +198,195 @@ function lovd_mapVariantToTranscripts (&$aVariant, $aTranscripts)
+        FROM ' . TABLE_GENOME_BUILDS
+)->fetchAllGroupAssoc();
+
+if (count($aActiveGBs) < 1) {
+    // Lift overs can only be performed if more than one GB is active.
+
+    foreach ($aActiveGBs as $sBuild => $aGBColumns) {
+        $aChr = $_DB->query(
+            'SELECT DISTINCT chromosome FROM ' . TABLE_VARIANTS .
+            ' WHERE ' . $aGBColumns['DNA'] . '= ""' . // DNA field of this GB is empty
+            '    AND mapping_flags & ' . MAPPING_ALLOW . // Mapping is allowed
+            '    AND NOT mapping_flags & ' . (MAPPING_NOT_RECOGNIZED | MAPPING_DONE) // Mapping is possible and necessary
+        )->fetchColumn();
+
+        if (!empty($aChr)) {
+            // There is more than one chromosome with missing variant description for this build.
+
+            // We pick a random chromosome to minimize the chances of multiple active users
+            //  activating lift overs on the same chromosome.
+            $sChr = array_rand($aChr);
+
+            $aVariantIDs = $_DB->query(
+                'SELECT id FROM ' . TABLE_VARIANTS .
+                ' WHERE chromosome = ' . $sChr . // Only taking our picked chromosome
+                '    AND ' . $aGBColumns['DNA'] . '= ""' . // DNA field of this GB is empty
+                '    AND mapping_flags & ' . MAPPING_ALLOW . // Mapping  is allowed
+                '    AND NOT mapping_flags & ' . (MAPPING_NOT_RECOGNIZED | MAPPING_DONE) . // Mapping is possible and necessary
+                ' LIMIT ' . $nMaxVariants
+            )->fetchColumn();
+
+            // We will loop through all variants that are missing DNA descriptions.
+            foreach ($aVariantIDs as $sVariantID) {
+
+                // This variant is now in progress, which we will indicate by mapping flags.
+                $_DB->query(
+                    'UPDATE ' . TABLE_VARIANTS .
+                    ' SET mapping_flags = mapping_flags | ' . MAPPING_IN_PROGRESS .
+                    ' WHERE id = ?', array($sVariantID)
+                );
+
+                list($bMappingTryAgain, $bMappingSuccessful) = array(false, false);
+
+                // Now we will loop through all alternative builds to try lift overs
+                //  from all possible references.
+                foreach ($aActiveGBs as $sSourceBuild => $aSourceGBColumns) {
+                    if ($sSourceBuild == $sBuild) {
+                        // We need to lift over TO $sBuild, so not FROM $sBuild.
+                        continue;
+                    }
+
+                    $sVariant = $_DB->query('SELECT VariantOnGenome/DNA FROM ' . TABLE_VARIANTS .
+                        ' WHERE id = ' . $sVariantID)->fetchColumn();
+                    if ($sVariant == '') {
+                        // Variant description is empty; no chance of lifting over from this build.
+                        continue;
+                    }
+
+                    $aVariant = lovd_getVariantInfo($sVariantID);
```

Comment:
I think you meant to put the DNA description there?

Reply:
Ah oopsie, you're right :)